lib/config/presets/internal/monorepo.ts

```diff
@@ -299,6 +308,20 @@ const patternGroups = {
   'aws-java-sdk-v2': '^software.amazon.awssdk:',
   babel6: '^babel6$',
   clarity: ['^@cds/', '^@clr/'],
+  'datatables-net-autofill': '^datatables.net-autofill',
+  'datatables-net-buttons': '^datatables.net-buttons',
+  'datatables-net-colreorder': '^datatables.net-colreorder',
+  'datatables-net-fixedcolumns': '^datatables.net-fixedcolumns',
+  'datatables-net-fixedheader': '^datatables.net-fixedheader',
+  'datatables-net-keytable': '^datatables.net-keytable',
+  'datatables-net-responsive': '^datatables.net-responsive',
+  'datatables-net-rowgroup': '^datatables.net-rowgroup',
+  'datatables-net-rowreorder': '^datatables.net-rowreorder',
+  'datatables-net-scroller': '^datatables.net-scroller',
+  'datatables-net-searchbuilder': '^datatables.net-searchbuilder',
+  'datatables-net-searchpanes': '^datatables.net-searchpanes',
+  'datatables-net-select': '^datatables.net-select',
+  'datatables-net-staterestore': '^datatables.net-staterestore',
```

Comment:
these should be grouped by source url too

Reply:
I did this in an attempt to limit the lines needed.  As these can be pulled in groups using regex.  Or else, we'll end up with (8 * 14 =) 112 lines in total to include all by `sourceUrl`.  In other words, they all have a unique `sourceUrl`.

The reason I used `repoGroup` for the base components is that I could not easily target them by a regex.

Reply:
we prefer to use source url, even if those are more lines.

we plan to auto group by source url in future, so we don't need this explicit monorepos by source url.